chart/prometheus-msteams/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "v1.4.1"
 description: A Helm chart for Kubernetes
 name: prometheus-msteams
 home: https://github.com/prometheus-msteams/prometheus-msteams
-version: 0.7.0
+version: 0.7.1
```

Comment:
I would consider this PR as a new feature, thus a minor change, not a patch change.